cluster/addons/pdcsi-driver/pd-csi-psp.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: csi-gce-pd-controller-psp
```

Comment:
Ideally if this is gce-pd it should be under https://github.com/kubernetes/kubernetes/tree/master/cluster/gce/addons rather than https://github.com/kubernetes/kubernetes/tree/master/cluster/addons